config.py

```diff
@@ -17,30 +17,32 @@
 """
 
 supervisor_base_prompt = """
-    You are an AI designed to check the work of other AIs. Below, you see a single question which a the AI was given, along with the AIs answer and explanation for that answer. 
+    You are an AI designed to check whether an answer is correct. Below, you see a question and an answer. The answer may be wrong. Your mission is to determine whether the answer is correct for the question, or not.
```

Comment:
I understand the logic here is to make the supervisor prompt the same as the evalutor prompt (or at least very similar). However, I'm worried that we haven't told the supervisor anything about an explanation and so it maybe "skips over" the explanation. Were you able to run any tests on GPT-4 or chatGPT to see if these new prompts still work? 

If they do, then I think there is nothing to change. However, if they don't then I think we should add a couple words in this prompt about an explanation.

Reply:
Good catch. We should include about the explanation in the prompt so the model knows to expect it below. I just pushed a new version of the prompt that's cleaned up and references the explanation.